src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -659,6 +657,19 @@ def __init__(
             "file": file,
         }
 
+        # Set extra requirements for the extractor, so they can be installed from docker
```

Comment:
```suggestion
        # Set extra requirements for the extractor, so they can be installed when using docker
```
or on. from would be you get the package from docker rather than install into. So you say into docker. but I think when using is probably best :)